wsmaster/che-core-api-factory-github/src/test/java/org/eclipse/che/api/factory/server/github/GithubFactoryParametersResolverTest.java

```diff
@@ -156,6 +161,52 @@ public void shouldReturnFactoryFromRepositoryWithDevfile() throws Exception {
     assertEquals(factoryUrlArgumentCaptor.getValue().getDevfileFilename(), "devfile.yaml");
   }
 
+  @Test
+  public void shouldSetDefaultProjectIntoDevfileIfNotSpecified() throws Exception {
```

Comment:
I'm missing a test for setting the branch from the URL into the generated project.

Reply:
i will expand this test